src/Draupnir.ts

```diff
@@ -298,17 +298,24 @@ export class Draupnir implements Client, MatrixAdaptorContext {
    * This means this is only used in the index.ts.
    */
   public async startupComplete(): Promise<void> {
-    const statusInfo = await draupnirStatusInfo(this);
     try {
       await this.managementRoomOutput.logMessage(
         LogLevel.INFO,
         "Draupnir@startup",
         "Startup complete. Now monitoring rooms."
       );
+      const statusInfo = await draupnirStatusInfo(this);
+      if (isError(statusInfo)) {
+        log.error(
+          `Was unable to get the status information for draupnir`,
+          statusInfo.error
+        );
+        return;
+      }
```

Comment:
I don't know if we need to do this anymore? since technically i think we got rid of the thing in `draupnirStatusInfo` that could fail